apps/src/blockly/addons/cdoFieldBitmap.ts

```diff
@@ -11,28 +22,67 @@ export class CdoFieldBitmap extends FieldBitmap {
    * @param {object | null} config - Additional configuration options, can be an object or null/undefined.
    */
   constructor(
-    value?: number[][] | null,
-    options?: object | null,
-    config?: {fieldHeight?: number} | null
+    value: number[][] | typeof Blockly.Field.SKIP_SETUP,
```

Comment:
nit: update the comment above to indicate the values can no longer be null (but can be undefined)